docs/guidelines/content-guidelines/01-user-docs.md

```diff
@@ -1,49 +1,77 @@
 # User Documentation
 
-Open-source Kyma and Kyma module user documentation is displayed at [`https://kyma-project.io`](https://kyma-project.io/#/). The website uses [docsify](https://docsify.js.org/#/) as a documentation site generator.
+Open-source Kyma and Kyma module user documentation is displayed at [`https://kyma-project.io`](https://kyma-project.io/#/). The website uses [VitePress](https://vitepress.dev/) as a documentation site generator.
 
 The overarching Kyma content is pulled from the `/docs` folder in the `/kyma` repository. Module documentation is pulled from the `/docs/user` folder in respective module repositories.
 
-After initialization, to run, docsify uses the following files located in the `/docs` folder in the `/kyma` repository:
+After initialization, to run, VitePress uses the following files in the `/kyma` repository:
 
-- `index.html` for configuration
-- `README.md` as the home page
+- `config.mjs` for configuration, including configuration of all sidebars 
+- `index.md` as the home page
+- `deploy.yml` to deploy documentation to the website
 - `.nojekyll` preventing GitHub Pages from ignoring files that begin with an underscore
-- `_sidebar.md` listing all Markdown documents to be displayed and ruling navigation between those documents on the website
 
 ## Publish a Document from the `/kyma` Repository
 
 To publish a document located in the `/kyma` repository, follow these steps:
 
 1. Create a pull request adding your content to a Markdown file located in the `/docs` folder.
-2. Add a new `_sidebar.md` file including a link to your document, or update an existing `_sidebar.md` to include it.
+2. Add a new `_sidebar.ts` file including a link to your document, or update an existing `_sidebar.ts` to include it.
+3. Run the [Deploy VitePress site to GitHub Pages](https://github.com/kyma-project/kyma/actions/workflows/deploy.yml) or wait for the CronJob to start it (every day at midnight).
+4. Make sure that the `public` folder is deleted after the build. If not, delete it manually to clean up the environment.
 
 ## Publish a Document from an Existing Module Repository
 
 To publish a document located in an existing module repository, follow these steps.
 
 1. Create a pull request adding your content to a Markdown file located in the `/docs/user` folder in your module repository.
-2. In the `docs/user` folder, add a new `_sidebar.md` file including a link to your document, or update the existing `_sidebar.md` to include it.
+2. In the `docs/user` folder, add a new `_sidebar.ts` file including a link to your document, or update the existing `_sidebar.ts` to include it.
```

Comment:
```suggestion
2. Update the `_sidebar.ts` file located in the `/docs/user` folder in the module repository, including a link to your document.
```
If this is an already existing repo then the sidebar.ts file should already exist.